### PR TITLE
Ft add logging

### LINKED ISSUE #1: Add Logging to bot
At this moment the bot doesn't have any logging. 
would be usefull to add logging

### PR DESCRIPTION
**Pull request by Darkguver**
add's logging output from the bot
fixes #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.idea
\ No newline at end of file
+.idea
+.env
```
Comment (yopefonic) [inline .gitignore]: new line ending needed for file

File: docker-compose.yaml [hunk 2]
```diff
@@ -6,4 +6,5 @@ services:
     restart: always
     environment:
       BOT_TOKEN: ${BOT_TOKEN}
-      VOICE_ROLE_ID: ${VOICE_ROLE_ID}
\ No newline at end of file
+      VOICE_ROLE_ID: ${VOICE_ROLE_ID}
+      LOG_LEVEL: ${LOG_LEVEL}
```
Comment (yopefonic) [inline docker-compose.yaml]: same here

File: src/FFB.py [hunk 3]
```diff
@@ -1,27 +1,42 @@
...
+            LOGGER.info(f"User:{member} left voice channel: {before.channel} removing Role")
             await member.remove_roles(role)
 
 intents = discord.Intents.default()
 intents.voice_states = True
 
+LOGGER = set_up_logging()
+
 client = MyClient(intents=intents)
 client.run(os.environ['BOT_TOKEN'])
```
Comment (yopefonic) [inline src/FFB.py]: same here

File: src/FFB.py [hunk 4]
```diff
@@ -1,27 +1,42 @@
...
 
 class MyClient(discord.Client):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     async def on_voice_state_update(self, member, before, after):
+        LOGGER.debug(f"following data for event happend: \n "
+                     f"User: {member} \n "
+                     f"State Before: {before} \n "
+                     f"State After:{after}")
```
Comment (yopefonic) [inline src/FFB.py:24]: Shouldn't there be an extra space? for formatting?

**Review by yopefonic — CHANGES_REQUESTED** [review]
beyond line ending and space LGTM

File: src/FFB.py (hunk 4 above)
Comment (Darkguver) [reply src/FFB.py:24]: Ran the pep8 test and this wasn't a issue ?

**Review by yopefonic — APPROVED** [review]

**Review by yopefonic — APPROVED** [review]

File: src/FFB.py (hunk 4 above)
Comment (yopefonic) [reply src/FFB.py:24]: it was meant more as a output styling issue ```f"State After: {after}")```

**Pull request merged**